app/assets/javascripts/components/data_portal/modals/modal-download-all.js

```diff
@@ -0,0 +1,83 @@
+(function (App) {
+  App.Component.ModalDownloadAll = App.Component.Modal.extend({
+
+    contentTemplate: JST['templates/data_portal/modals/modal-download-all'],
+
+    defaults: {
+      // See App.Component.Modal for details about this option
+      title: 'Download All',
+      // See App.Component.Modal for details about this option
+      showTitle: true,
+      // See App.Component.Modal for details about this option
+      footer: '<div class="group-button"><button class="c-button -white js-close">Close</button></div><div class="group-button"><a disabled href="#" download class="c-button -sea js-continue">Continue</a></div>'
```

Comment:
We have an issue here. While disabled buttons are avoided when using keyboard navigation, disabled links aren't. This means that the user can still focus on the button and download the file without having to enter its email address.

I think the easiest solution would be to use a real button, and to make the page download the file by creating an artificial link and clicking on it.

This would be added to `_onContinue`:

```js
var link = document.createElement('a');
link.setAttribute('download');
link.href = 'YOUR LINK HERE';
link.click();
```

Please make sure to test it with VoiceOver.